File: Exemplos/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid/IntentServices/Bridge/ImprimirCupomSatCancelamento.cs
namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Bridge
{
    class ImprimirCupomSatCancelamento : BridgeCommand
    {
        readonly string xml;
        readonly string assQRCode;

        public ImprimirCupomSatCancelamento(string xml, string assQRCode) : base("ImprimirCupomSatCancelamento")
        {
            this.xml = xml;
            this.assQRCode = assQRCode;
        }

        protected override string FunctionParameters()
        {
            return "\"xml\"" + ":" + "\"" + xml + "\"" + "," +
                    "\"assQRCode\"" + ":" + "\"" + assQRCode + "\"";
        }
    }
}

File: Exemplos M10/App_IntentDigitalHUB_ReactNative/Elgin_IntentDigitalHUB_ReactNative-font/src/intentDigitalHubService/BRIDGE/ImprimirCupomSatCancelamento.ts
import {BridgeCommand} from './BridgeCommand';

export class ImprimirCupomSatCancelamento extends BridgeCommand {
  readonly xml: string;
  readonly assQRCode: string;

  constructor(xml: string, assQRCode: string) {
    super('ImprimirCupomSatCancelamento');
    this.xml = xml;
    this.assQRCode = assQRCode;
  }

  functionParameters(): object {
    return {xml: this.xml, assQRCode: this.assQRCode};
  }
}
